src/components/admin/WaitListCard.tsx

```diff
@@ -10,6 +9,7 @@ interface ApplicantProps {
   getUserById: () => void;
 }
 
+
 const WaitListCard: React.FC<ApplicantProps> = ({
```

Comment:
you can wrap the onSelect and getUserById using a useCallback hook to avoid unnecessary re-renders